internal/pkg/describe/env.go

```diff
@@ -246,6 +253,27 @@ func (d *EnvDescriber) filterDeployedSvcs() ([]*config.Workload, error) {
 	return deployedSvcs, nil
 }
 
+// filterDeployedJobs lists the jobs that are deployed on the given app and environment
```

Comment:
nit: it's nicee to have a description but function comment is not required if the function is not exported.